mood-analysis.rb

```diff
@@ -28,5 +86,16 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+stripped = []
+
+text.each do |string|
+  stripped << strip_punctuation(string)
+end
```

Comment:
If you had decided to call strip_punctuation inside the analyze_mood method, you wouldn't have had to do this code so repetitively throughout. What I wanted you to think about is "Is there ever a situation where I wouldn't want to strip the punctuation?" and if there isn't then this should be built into the analyze_mood method.